### PR TITLE
OSDOCS-3554: Adding info on restricted-v2 SCC

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3554

Link to docs preview:
http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3554-auth/migrating_from_ocp_3_to_4/planning-migration-3-4.html#migration-preparing-security

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@stlaz @y4sht Can you please review?

Preview: http://file.rdu.redhat.com/~ahoffer/2022/OSDOCS-3554-auth/migrating_from_ocp_3_to_4/planning-migration-3-4.html#migration-preparing-security

**Review by kelbrown20 — COMMENTED** [review]
This looks great! LGTM!

File: migrating_from_ocp_3_to_4/planning-migration-3-4.adoc [hunk 1]
```diff
@@ -189,6 +189,13 @@ For more information, see xref:../authentication/understanding-identity-provider
 
 Newly created OAuth HTTP bearer tokens no longer match the names of their OAuth access token objects. The object names are now a hash of the bearer token and are no longer sensitive. This reduces the risk of leaking sensitive information.
 
+[discrete]
+==== Default security context constraints
+
+The `restricted` security context constraints (SCC) in {product-title} 4 no longer has the same authenticated access as the `restricted` SCC in {product-title} 3.11. The `restricted-v2` SCC is now the equivalent {product-title} 4 SCC and replaces the `restricted` SCC for new clusters.
```
[the remark this answers is not in the document]
Comment (bergerhoffer) [reply migrating_from_ocp_3_to_4/planning-migration-3-4.adoc]: @stlaz Thanks! Incorporated your wording with some minor adjustments.

**Comment by y4sht** [comment]
LGTM

**Pull request merged**